src/fusedav.c

```diff
@@ -754,7 +755,10 @@ static void get_stat(const char *path, struct stat *stbuf, GError **gerr) {
      * if we failed to reset it correctly.
      */
     // If the parent directory is out of date, update it.
```

Comment:
Should this comment remain above the if-statement?

Reply:
Keep your eyes open. It just might change position in the next update ;)